src/components/Business.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" src="./business.ts" />
+
+<template>
+    <div>
+        <span class="info">Équipe : </span>
+        <button @click="buttonFilter='principal'">Principale</button> <button @click="buttonFilter='all'">Tous</button>
+        <div class="profils">
```

Comment:
[https://alticreation.com/bem-pour-le-css/](https://alticreation.com/bem-pour-le-css/
)
```suggestion
        <div class="business__profiles">
```